containerimages/BUILD.bazel

```diff
@@ -134,3 +134,15 @@ container_image(
     mode = "444",
     visibility = ["//visibility:public"],
 )
+
+container_image(
```

Comment:
Is this additinal layer just added  for consistency?

Reply:
yes